File: RobocodeLabo/src/laboratorio/StrategyJuniorRobot.java
package laboratorio;

import robocode.JuniorRobot;

public interface StrategyJuniorRobot {

	void run(JuniorRobot aJuniorRobot);

	void onHitWall(JuniorRobot aJuniorRobot);

	void onHitByBullet(JuniorRobot aJuniorRobot);

	void onScannedRobot(JuniorRobot aJuniorRobot);

}

File: Entrega-2-kt/src/laboratorio/StrategyJuniorRobot.kt
package laboratorio

import robocode.JuniorRobot

interface StrategyJuniorRobot {
    fun run(aJuniorRobot: JuniorRobot)
    fun onScannedRobot(aJuniorRobot: JuniorRobot)
    fun onHitByBullet(aJuniorRobot: JuniorRobot)
    fun onHitWall(aJuniorRobot: JuniorRobot)

}